src/main/java/com/example/parkingapp/api/rest/vm/VehicleStatusVM.java

```diff
@@ -0,0 +1,97 @@
+package com.example.parkingapp.api.rest.vm;
+
+import com.example.parkingapp.config.AcceptedCurrencies;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+/**
+ * View Model object for storing parked vehicle status
+ */
+public class VehicleStatusVM {
+    private String licensePlate;
+    private LocalDateTime startDate;
+    private LocalDateTime endDate;
+    private long parkingTimeInHours;
+    private BigDecimal parkingCost;
```

Comment:
plus za pieniądze w BigDecimal - ale patrzyłeś jak one się wyświetlają w jsonie?

Reply:
Wyświetlają się jako number, żadne „dziwne” pola się nie serializują w tym przypadku (potencjalny problem to strata precyzji i wyświetlanie zbyt dużo miejsc po przecinku)